nodes/nomos-node-api/src/http/backend/axum.rs

```diff
@@ -1,17 +1,19 @@
-use std::{fmt::Debug, hash::Hash, net::SocketAddr, sync::Arc};
+use std::{fmt::Debug, hash::Hash, net::SocketAddr};
 
-use axum::{extract::State, response::IntoResponse, routing, Json, Router, Server};
-use full_replication::Blob;
-use hyper::StatusCode;
+use axum::{extract::State, response::Response, routing, Json, Router, Server};
+use consensus_engine::BlockId;
+use full_replication::{Blob, Certificate};
 use nomos_core::{da::blob, tx::Transaction};
-use nomos_mempool::{openapi::Status, MempoolMetrics};
+use nomos_mempool::{network::adapters::libp2p::Libp2pAdapter, openapi::Status, MempoolMetrics};
+use nomos_network::backends::libp2p::Libp2p;
+use nomos_storage::backends::StorageSerde;
 use overwatch_rs::overwatch::handle::OverwatchHandle;
-use serde::{Deserialize, Serialize};
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use utoipa::OpenApi;
 use utoipa_swagger_ui::SwaggerUi;
 
 use crate::{
-    http::{cl::*, da::*},
+    http::{cl, consensus, da, libp2p, mempool, storage},
     Backend,
 };
```

Comment:
Nitpick. All imports are mixed. We should try to separate in blocks:

```rust
// std

// crates

// internal
```

Reply:
Sorry, I am a little bit confused here, do you mean we should treat `nomos*` as internal crates? Otherwise, it seems imports are already in the std - crates - internal format. 

```rust
use std::{fmt::Debug, hash::Hash, net::SocketAddr};

use axum::{extract::State, response::Response, routing, Json, Router, Server};
use consensus_engine::BlockId;
use full_replication::{Blob, Certificate};
use nomos_core::{da::blob, tx::Transaction};
use nomos_mempool::{network::adapters::libp2p::Libp2pAdapter, openapi::Status, MempoolMetrics};
use nomos_network::backends::libp2p::Libp2p;
use nomos_storage::backends::StorageSerde;
use overwatch_rs::overwatch::handle::OverwatchHandle;
use serde::{de::DeserializeOwned, Deserialize, Serialize};
use utoipa::OpenApi;
use utoipa_swagger_ui::SwaggerUi;

use crate::{
    http::{cl, consensus, da, libp2p, mempool, storage},
    Backend,
};
```

Reply:
Yes, anything internally related to the project I think is better in internal:
* consensus_engine
* nomos_core
* nomos_mempool
etc

Overwatch can be kept in crates 👍